Cargo.toml

```diff
@@ -67,4 +67,4 @@ pr-run-mode = "skip"
 # The profile that 'cargo dist' will build with
 [profile.dist]
 inherits = "release"
-lto = "thin"
+lto = "thin"
```

Comment:
I might be reading this diff wrong, but can you make sure there is a newline at the end?

Reply:
Done!